proposals/lip-0060.md

```diff
@@ -31,13 +31,13 @@ The new genesis block format and processing is specified with the rationale of h
 
 ### Processing of the genesis block
 
-Another distinctive specification of the genesis block is its processing. The blockchain starts with an empty key-value store and in particular, all module stores are initially empty. As part of the genesis block processing modules can add key-value entries to their module store to initialize their state. They should also check the consistency of their state independently and against information provided by other modules. Also, before this is done, the framework layer has to validate the block header and the block format itself. Hence, this LIP defines steps specific to the processing of the genesis block: a step to initialize the state store per each module and the second step to verify this initial state. In particular the processing of the genesis block happens in four steps described in the sections below.
+Another distinctive specification of the genesis block is its processing. The blockchain starts with an empty key-value store and in particular, all module stores are initially empty. As part of the genesis block processing modules can add key-value entries to their module store to initialize their state. They should also check the consistency of their state independently and against information provided by other modules. Also, before this is done, the framework layer has to validate the block header and the block format itself. Hence, this LIP defines steps specific to the processing of the genesis block: a step to initialize the state store per each module and the second step to verify this initial state. In particular the processing of the genesis block happens in five steps described in the sections below.
```

Comment:
> in five steps described in the sections below.

not a big deal: the diagram visualizes 6 steps. Also tagging @ricott1. Feel free to ignore this comment.

Reply:
I removed the exact number of steps.